### PR TITLE
Feat: add package can include binaries

### PR DESCRIPTION
**Pull request by aryan-rajoria**
Looks for binary names in metadata provided from crates.io.

### CODE REVIEW

File: depscan/lib/package_query/cargo_pkg.py [hunk 1]
```diff
@@ -112,5 +120,7 @@ def cargo_pkg_risk(pkg_metadata, is_private_pkg, scope, pkg):
         risk_metrics[f"pkg_{scope}_scope_risk"] = True
         risk_metrics[f"pkg_{scope}_scope_value"] = 1
 
+    risk_metrics['pkg_includes_binary_risk'] = does_binary_exist(versions_dict.get(pkg.get("version"), {}))
```
Comment (prabhu) [inline depscan/lib/package_query/cargo_pkg.py:123]: We also need a value based on the number of binaries

Comment (prabhu) [reply depscan/lib/package_query/cargo_pkg.py:123]: We also handle the cases where we check the latest version if no version number is specified?

Comment (aryan-rajoria) [reply depscan/lib/package_query/cargo_pkg.py]: added both of these changes

File: depscan/lib/package_query/cargo_pkg.py [hunk 2]
```diff
@@ -57,31 +60,32 @@ def cargo_pkg_risk(pkg_metadata, is_private_pkg, scope, pkg):
...
-            version_nums
-        )
-    except (ValueError, TypeError):
-        # First version number is latest, while last is the oldest release.
-        first_version_num = version_nums[-1]
-        latest_version_num = version_nums[0]
-    first_version = versions_dict.get(first_version_num)
-    latest_version = versions_dict.get(latest_version_num)
+
+    # Logic fails for version like 2.0.9 > 2.0.14 in python
```
Comment (prabhu) [inline depscan/lib/package_query/cargo_pkg.py:64]: Can you use semver comparison? Example: https://github.com/owasp-dep-scan/dep-scan/blob/master/depscan/lib/package_query/pypi_pkg.py#L58

Comment (aryan-rajoria) [reply depscan/lib/package_query/cargo_pkg.py:64]: cargo api does sort it with 0 being the latest and the last being the oldest version.
should I still add the logic?

Comment (prabhu) [reply depscan/lib/package_query/cargo_pkg.py:64]: Yes, good idea to use our logic since sorting a sorted list will be faster

**Review by prabhu — APPROVED** [review]
Thank you so much!

**Pull request merged**